djstripe/migrations/0005_auto_20190703_0958.py

```diff
@@ -0,0 +1,88 @@
+# Generated by Django 2.2.2 on 2019-07-03 09:58
+
+from django.db import migrations, models
+import django.db.models.deletion
+import djstripe.enums
+import djstripe.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('djstripe', '0004_auto_20190612_0850'),
+    ]
+
+    operations = [
+		# Altering the index is required because while we chnage the name of old PaymentMethod model to
+		# DjStripePaymentMethod, the migrations didn't update the names of index.
+		# In the current migration, we create a new PaymentMethod model, hence before creating it, its
+		# better to rename the old index.
+		migrations.RunSQL(
+			'ALTER INDEX djstripe_paymentmethod_id_0b9251df_like rename TO djstripe_paymentmethod_legacy_id_0b9251df_like',
+			'ALTER INDEX djstripe_paymentmethod_legacy_id_0b9251df_like rename TO djstripe_paymentmethod_id_0b9251df_like',
+		),
+		migrations.AlterField(
+            model_name='payout',
+            name='status',
+            field=djstripe.fields.StripeEnumField(enum=djstripe.enums.PayoutStatus, help_text='Current status of the payout. A payout will be `pending` until it is submitted to the bank, at which point it becomes `in_transit`. It will then change to paid if the transaction goes through. If it does not go through successfully, its status will change to `failed` or `canceled`.', max_length=10),
```

Comment:
If this alterfield is just changing help_text it can be done by editting the original migration 

(Not a huge problem here since we're adding a migration anyway, but it means typo fixes can potentially be backported to stable branches)